openlibrary/plugins/openlibrary/dev_instance.py

```diff
@@ -160,3 +177,79 @@ def load_sample_data():
     comment = "Loaded sample data."
     list_key = "/people/anand/lists/OL1815L"
     env['copy_list'](src, dest, list_key, comment=comment)
+
+
+def start_debugger():
+    """ Attach debugger and immediately break into debugger """
+
+    logger.debug("------------> ATTEMPTING TO ENABLE DEBUGGING <--------------")
+    try:
+        sys.path.append("/usr/local/lib/python2.7/dist-packages")
+        ptvsd = import_module("ptvsd")
+        ptvsd.enable_attach(address=('0.0.0.0', 3000))
+        ptvsd.wait_for_attach()
+        ptvsd.break_into_debugger()
+        sys.path.pop()
+    except Exception as e:
+        logger.error("something bad happened {}".format(str(e)))
+    logger.debug("-----------------> DEBUGGING ENABLED <----------------------")
+
+
+def permission(key, readers, writers, admins):
+    """ Query for permission type \"thing\"s """
```

Comment:
These backslashes are not required and confusing.

Reply:
changing to use reST (restructured text) and removing the escaped `thing`